app/code/Magento/Quote/Api/CartItemRepositoryInterface.php

```diff
@@ -22,7 +22,18 @@ interface CartItemRepositoryInterface
     public function getList($cartId);
 
     /**
-     * Add/update the specified cart item.
+     * Add the specified cart item.
+     *
+     * @param \Magento\Quote\Api\Data\CartItemInterface $cartItem The item.
+     * @return \Magento\Quote\Api\Data\CartItemInterface Item.
+     * @throws \Magento\Framework\Exception\NoSuchEntityException The specified cart does not exist.
+     * @throws \Magento\Framework\Exception\CouldNotSaveException The specified item could not be saved to the cart.
+     * @throws \Magento\Framework\Exception\InputException The specified item or cart is not valid.
+     */
+    public function add(\Magento\Quote\Api\Data\CartItemInterface $cartItem);
+
+    /**
+     * Update the specified cart item.
```

Comment:
Let's leave the `CartItemRepositoryInterface` as is, let's add two more interfaces, one to handle `add` and another one to handle `update`. Also, let's mark the `save` method as deprecated and reference the other two interfaces.